src/app/dataset-view/dataset.service.ts

```diff
@@ -274,9 +274,13 @@ export class DatasetService {
             const metadataSchemaUpdate: MetadataSchemaUpdate = {
                 schema,
                 pageInfo,
-                metadataSummary: metadata,
+                metadataSummary,
             };
             this.datasetSubsService.emitMetadataSchemaChanged(metadataSchemaUpdate);
+
+            this.datasetSubsService.emitCurrentTransformInputsCountChanged(
+                metadataSummary.metadata?.currentTransform?.inputs.length ?? 0,
+            );
```

Comment:
Any component can write a derived observable like this based on the main topic.

Reply:
Removed